cypher/cypher-docs/src/docs/dev/glossary.asciidoc

```diff
@@ -270,3 +272,70 @@ This section comprises a glossary of all the keywords -- grouped by category and
+|===
+
+[[glossary-privileges]]
+=== Privileges
+
+[options="header"]
+|===
+|Name                                                                                           | Category              | Description
+| <<administration-security-administration-database-access, ACCESS>>                            | Database              | Determines whether a user can access a specific database.
+| <<administration-security-administration-database-startstop, START>>                          | Database              | Determines whether a user can start up a specific database.
+| <<administration-security-administration-database-startstop, STOP>>                           | Database              | Determines whether a user can stop a specific running database.
+| <<administration-security-administration-database-indexes, CREATE INDEX>>                     | Schema                | Determines whether a user is allowed to create indexes on a specific database.
+| <<administration-security-administration-database-indexes, DROP INDEX>>                       | Schema                | Determines whether a user is allowed to drop indexes on a specific database.
+| <<administration-security-administration-database-indexes, INDEX MANAGEMENT>>                 | Schema                | Determines whether a user is allowed to create and drop indexes on a specific database.
+| <<administration-security-administration-database-constraints, CREATE CONSTRAINT>>            | Schema                | Determines whether a user is allowed to create constraints on a specific database.
+| <<administration-security-administration-database-constraints, DROP CONSTRAINT>>              | Schema                | Determines whether a user is allowed to drop constraints on a specific database.
+| <<administration-security-administration-database-constraints, CONSTRAINT MANAGEMENT>>        | Schema                | Determines whether a user is allowed to create and drop constraints on a specific database.
+| <<administration-security-administration-database-tokens, CREATE NEW NODE LABEL>>             | Schema                | Determines whether a user is allowed to create new node labels on a specific database.
+| <<administration-security-administration-database-tokens, CREATE NEW RELATIONSHIP TYPE>>      | Schema                | Determines whether a user is allowed to create new relationship types on a specific database.
+| <<administration-security-administration-database-tokens, CREATE NEW PROPERTY NAME>>          | Schema                | Determines whether a user is allowed to create new property names on a specific database.
+| <<administration-security-administration-database-tokens, NAME MANAGEMENT>>                   | Schema                | Determines whether a user is allowed to create new labels, types and property names on a specific database.
```

Comment:
These are Token privileges not Schema

Reply:
"Token" is a word we use internally but it is not used in any documentation. So I am not in favor of introducing it here.

Reply:
So user NAME? The command is NAME MANAGEMENT (which I don't think falls under the SCHEMA MANAGEMENT in Ivans document (I think he might call it token there but not sure)

Reply:
Still standing by my schema argument. The name of things are part of the schema in a conceptual way. Its literally what you get from calling db.schema():  the name of things (and how they are related). 
Ivans document is sadly not part of the documentation, so lets not use internal words and concepts that are not explained in the docs.